cabal.project

```diff
@@ -3,3 +3,18 @@
 packages:
     ./
     , ./example
+
+constraints:
+    persistent == 2.14.1.0
+
+tests: True
+
+allow-newer:
+    http-api-data:base
+    , persistent:aeson
+
+-- https://github.com/bennofs/th-lift-instances/pull/33
+source-repository-package
+    type: git
+    location: https://github.com/TeofilC/th-lift-instances
+    tag: bd768ec924d7c812bcc86e6f24fd31ba7e310f5a
```

Comment:
I can go ahead and release this one. Users will get the build failure with `th-lift-instances`, but whenever that comes out with a new one, we'll be set.